bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "adapt-contrib-matching",
-  "version": "6.0.0",
+  "version": "6.0.2",
```

Comment:
Probably should remove this version change. It's usually handled at release time